src/reanimated2/core.ts

```diff
@@ -232,3 +232,8 @@ export function configureProps(uiProps: string[], nativeProps: string[]): void {
     NativeReanimatedModule.configureProps(uiProps, nativeProps);
   }
 }
+
+if (!global.setImmediate) {
```

Comment:
I agree with Tomek 👍

Reply:
This is a side effect for web, and needs to get added to the `global` side effects-friendly file: https://github.com/software-mansion/react-native-reanimated/blob/1ec1a0f6115e49a2f5d9481264711023fde0bf11/src/reanimated2/js-reanimated/global.ts

Otherwise, webpack can tree shake this out.

Reply:
It looks like `src/reanimated2/core.ts` is also included in `sideEffects`:

https://github.com/software-mansion/react-native-reanimated/blob/870fe898498c99032e81d75a52c53ae407aae86a/package.json#L174-L178

But yeah, I agree it would be nice to have it in `global.ts`.

Reply:
oh, got it, should be fine then

Reply:
(as long as the full path to that file is marked as a side effect)